compiler/rustc_query_system/src/dep_graph/graph.rs

```diff
@@ -582,23 +594,58 @@ impl<K: DepKind> DepGraph<K> {
 
     #[inline]
     pub fn dep_node_index_of_opt(&self, dep_node: &DepNode<K>) -> Option<DepNodeIndex> {
-        let data = self.data.as_ref().unwrap();
-        let current = &data.current;
+        self.data.as_ref().unwrap().dep_node_index_of_opt(dep_node)
```

Comment:
maybe get rid of the unwrap since we're modifying this line anyways (if it's simple to do so)

Reply:
Could we get rid of this method?

Reply:
Done.